seed/serializers/analyses.py

```diff
@@ -9,8 +9,24 @@
 from seed.models import Analysis
 
 
+class CustomChoicesField(serializers.ChoiceField):
+    def to_representation(self, obj):
+        if obj == '' and self.allow_blank:
+            return obj
+        return self._choices[obj]
+
+    def to_internal_value(self, data):
+        if data == '' and self.allow_blank:
+            return ''
+
+        for key, val in self._choices.items():
+            if val == data:
+                return key
+        self.fail('invalid_choice', input=data)
+
+
 class AnalysisSerializer(serializers.ModelSerializer):
-    service = serializers.CharField(source='get_service_display')
+    service = CustomChoicesField(Analysis.SERVICE_TYPES)
```

Comment:
I think previously, we let the frontend handle the translation from enum to string, but I’m fine with this. And if others disagree, it's not a significant effort to make the change at some point.

Reply:
As a quick comment to why I did this, it seemed like the easiest way to deserialize (ie from json to python) when a user posts from the FE, now the FE just needs to know the "display" names.